extensions/omni_manifest/docs/usage.md

```diff
@@ -242,4 +242,75 @@ installed, at which point it'll return `kept`.
 !!! tip  "Install transactionally"
 
     It is highly recommended to perform installations transactionally to be able to check
-    reports for missing dependencies.
\ No newline at end of file
+    reports for missing dependencies.
+
+## Update extensions
+
+To update extensions using `omni_manifest.install` ensure the upgrade scripts for the extensions is saved to the postgres extension
+scripts directory. Currently all the extensions are versioned using git commit sha of omnigres repo, extension upgrade is only possible from older commit version to newer ones.
+
+```postgresql
+-- check available version of omni_python
+select * from pg_available_extension_versions where name = 'omni_python';
+    name     | version | installed | superuser | trusted | relocatable |   schema    |   requires   | comment 
+-------------+---------+-----------+-----------+---------+-------------+-------------+--------------+---------
+ omni_python | 2c7e737 | f         | t         | f       | f           | omni_python | {plpython3u} | 
+(1 row) 
+
+-- create omni_python extension with version '2c7e737'
+select *                                
+from
+    omni_manifest.install(
+            'omni_python=2c7e737#plpython3u=*'::text::omni_manifest.artifact[]
+    );
+      requirement      |  status   
+-----------------------+-----------
+ (plpython3u,*)        | installed
+ (omni_python,2c7e737) | installed
+(2 rows)
+```
+
+A public s3 endpoint is available to download extension upgrade scripts (old to new versions)
```

Comment:
Perhaps worth mentioning here that this applies to Omnigres-built extensions only?

Reply:
done